packages/Select/src/Select.tsx

```diff
@@ -0,0 +1,256 @@
+import * as React from 'react';
```

Comment:
is it possible to combine all the select.tsx into one? It just seems to get kind of confusing looking at all these. It seems like maybe we're adding more code than we need to. Or maybe it's just me.

Reply:
I personally prefer to separate many components in many files (1 component by file).